src/Workspaces/Core/Desktop/Workspace/SolutionSize/SolutionSizeTracker.cs

```diff
@@ -32,12 +30,16 @@ internal class SolutionSizeTracker : IIncrementalAnalyzerProvider
         /// </summary>
         public long GetSolutionSize(Workspace workspace, SolutionId solutionId)
         {
-            return workspace is VisualStudioWorkspaceImpl ? _tracker.GetSolutionSize(solutionId) : -1;
+            var service = workspace.Services.GetService<IPersistentStorageLocationService>();
```

Comment:
do we want this in workspace layer or leave it in VS layer? with this in workspace layer, now roslyn used in other place such as ETA will start to use esent.